classes/person.rb

```diff
@@ -2,13 +2,14 @@
 # Base class for all persons
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..10_000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
**You did a great job!**

- [ ] I can see an initializer for rentals however, for adding rentals I was not able to find a method in your person. rb. There should be a method like that to add rentals:


```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
``` 
kindly add this method.

Reply:
I added the methods without date argument since it is autogenerated

Reply:
Hi @gdumani, But if you watch the given demo in requirement. You can see the data is being added manually.

Reply:
A big picture of the project: [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/sneak_peek.md)

Reply:
A video: [link](https://www.youtube.com/watch?v=vkkgrhD6aXQ&ab_channel=EmiliaAndrzejewska)